### DISCUSSION TITLE
CHAOSS Metrics 2022-04 release

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#36] [2️⃣ 聊项目]
[CHAOSS community](https://chaoss.community/metrics/) new version of Metrics, and included in our knowledge base~

- English version：https://xlab2017.yuque.com/msdpvs/rpifg5/26124511
- Chinese version：https://xlab2017.yuque.com/msdpvs/rpifg5/26124512

**Comment by frank-zsy** [comment]
看起来很多指标确实都只能用调研的方式落地了，问卷研究的框架确实非常重要。

另外，GitHub 对学术还是非常友好的，可以给项目添加引用的配置文件，这样可以在论文中直接对开源项目进行引用，这个也是很有用的：https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files

**Reply by will-ww** [reply]
> 看起来很多指标确实都只能用调研的方式落地了，问卷研究的框架确实非常重要。
> 
> 另外，GitHub 对学术还是非常友好的，可以给项目添加引用的配置文件，这样可以在论文中直接对开源项目进行引用，这个也是很有用的：https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files

是的，这块可以专业的做起来，借助工具和平台，也是个很好的积累点~
